jsdom/jsdom.ts

```diff
@@ -0,0 +1,9 @@
+import "./vendor/jsdom.js";
+
+let JSDOM = (window as any).jsdom.JSDOM;
+let dom = new JSDOM(`
+  <!DOCTYPE html>
+  <p>Hello, Deno!</p>
+`);
```

Comment:
Cool! How big is the vendor/jsdom.js file?

Reply:
3.05 MiB

I added a TODO for investigating whether we can reduce this size.